doc/classes/HTTPRequest.xml

```diff
@@ -149,8 +149,8 @@
 		}
 		[/csharp]
 		[/codeblocks]
-
 		[b]Gzipped response bodies[/b]: HTTPRequest will automatically handle decompression of response bodies. A [code]Accept-Encoding[/code] header will be automatically added to each of your requests, unless one is already specified. Any response with a [code]Content-Encoding: gzip[/code] header will automatically be decompressed and delivered to you as uncompressed bytes.
+		[b]Note:[/b] When exporting to Android, make sure to enable the [code]INTERNET[/code] permission in the Android export preset before exporting the project or using one-click deploy. Otherwise, network communication of any kind will be blocked by Android.
```

Comment:
In https://github.com/godotengine/godot/pull/51513, I just removed two other notes here and added a link to HTTPClient at the top and pointed to that warnings instead, both to avoid duplication and because on HTTPRequest, these warnings end up down very low on the page due do the code examples.

So IMO, its enough to add this to HTTPClient?

Reply:
I'm not sure about this. While SSL-related notes are secondary to most users, a lot of people are exporting to Android and are struggling with the process.

Reply:
I can see that, I guess. Okay. :) Maybe move it to the top, below the Warning one then? It's really easily missed down there if you look at the docs page.